reference/Chronicler.v0.yaml

```diff
@@ -0,0 +1,600 @@
+openapi: 3.1.0
+x-stoplight:
+  id: 4nnx31zof0m9d
+info:
+  title: Chronicler.v0
```

Comment:
Again, I'd title this "Chronicler 2" and use the version field below to indicate version 0.

Reply:
Considering that at some point this API will have a major bump - I wanted to check if it makes sense to nest the other APIs in a folder. I'm unsure how this would work with Stoplight - but the way the naming currently is might end up looking misleading.

Reply:
The problem I find there is that it can lead to confusion. There's already a Chronicler v2 - having Chronicler 2 in addition to that wouldn't work in my eyes. Alternatively, we revise the naming so that the title doesn't include the version of the API.

Reply:
thanks for renaming. At the moment, for better or worse the project is called "Chron 2" and like it or not, "Chron 2" is in version 0, and might some day make it to version 1 or version 2 itself. It isn't like we're just gonna skip around unused numbers, jumping from v0 to v3. If you don't want to name it chron 2, convince Astrid to rename the project.

Reply:
I'm just saying that it'll lead to some confusion. As for renaming, I meant that the title of Chronicler v2 would just be Chronicler - that way we avoid that confusion and differentiate between the two by version. I'll merge for now but yeah.

Reply:
That was mildly my point, we might sadly never be able to differentiate by version. Unless the new project is renamed, we might some day have both a Chronicler (v2) and a Chronicler 2 (v2).